docs/MCP_SSE.md

```
@@ -105,39 +170,80 @@ Example:
+
+    Note over Client,DB: Optional SSE Stream
+    Client->>MCP: GET /mcp (Accept: text/event-stream)
+    MCP-->>Client: SSE stream for server notifications
+```
 
 ### Authentication
 
 **Header-based** (preferred when supported):
+
 ```
 Authorization: Bearer <AUTOMEM_API_TOKEN>
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)